### PR TITLE
print/fontforge:20201107

### PR DESCRIPTION
**Pull request by yzgyyang**
```
print/fontforge: Update to 20201107

- Update WWW
- Move build from autotools to CMake

Changelogs:

- https://github.com/fontforge/fontforge/releases/tag/20200314
- https://github.com/fontforge/fontforge/releases/tag/20201107

PR: 252343
Submitted by: Naram Qashat <cyberbotx@cyberbotx.com> (maintainer)
Approved by: lwhsu
```

### CODE REVIEW

**Comment by lwhsu** [comment]
The submitter has change log attached, it's good to also attach them:

Changelogs:

https://github.com/fontforge/fontforge/releases/tag/20200314
https://github.com/fontforge/fontforge/releases/tag/20201107

File: patches/print.fontforge.20201107.diff [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+diff --git a/print/fontforge/pkg-descr b/print/fontforge/pkg-descr
+index 5286247bb570..9fd86d3c2125 100644
+--- a/print/fontforge/pkg-descr
++++ b/print/fontforge/pkg-descr
+@@ -12,4 +12,4 @@ from imported bitmap images.
+ 
+ For more information, see
+ 
+-WWW: http://fontforge.sourceforge.net/
++WWW: https://fontforge.org/en-US/
```
Comment (lwhsu) [inline patches/print.fontforge.20201107.diff:242]: I think we can just use https://fontforge.org

**Review by lwhsu — APPROVED** [review]

File: patches/print.fontforge.20201107.diff (hunk 1 above)
Comment (yzgyyang) [reply patches/print.fontforge.20201107.diff:242]: Well, it gets redirected anyways ;)

**Comment by yzgyyang** [comment]
> The submitter has change log attached, it's good to also attach them:
> 
> Changelogs:
> 
> https://github.com/fontforge/fontforge/releases/tag/20200314
> https://github.com/fontforge/fontforge/releases/tag/20201107

Updated.

File: patches/print.fontforge.20201107.diff (hunk 1 above)
Comment (lwhsu) [reply patches/print.fontforge.20201107.diff:242]: Yes but it looks more formal. :-)

Comment (yzgyyang) [reply patches/print.fontforge.20201107.diff:242]: Okay I guess you have a point, will make the switch ;)

**Pull request merged**